pywemo/subscribe.py

```diff
@@ -440,13 +467,28 @@ def sleep(secs: float) -> None:
 
         self._http_thread: threading.Thread | None = None
         self._httpd: HTTPServer | None = None
+        self._requested_port: int | None = requested_port
+        self._callback_address: str | None = callback_address
+        self._callback_port: int | None = callback_port
 
     @property
     def port(self) -> int:
         """Return the port that the http server is listening on."""
         assert self._httpd
         return self._httpd.server_address[1]
 
+    @property
+    def callback_port(self) -> int:
```

Comment:
I think we don't need `callback_port` anymore. Clients (the Subscription & ssdp.DiscoveryResponder) can use `.port`.

`get_callback_address` can translate to the correct IP & port for the callback when `PYWEMO_CALLBACK_ADDRESS` is supplied.

That way
* PYWEMO_CALLBACK_PORT is used internally for the local port
* PYWEMO_CALLBACK_ADDRESS provides the IP and port to the WeMo's callback parameter.